page-objects/pages/TASKListPage.js

```diff
@@ -31,8 +31,8 @@ const taskActionMenuButton = {
 	archive: "//*[text()[contains(.,'Abschließen')]]",
 	unarchive: "//*[text()[contains(.,'Wiederherstellen')]]",
 	edit: "//*[text()[contains(.,'Bearbeiten')]]",
-	copy: '.fa-copy',
-	delete: '.btn-delete',
+	copy: "//*[text()[contains(.,'Kopieren')]]",
+	delete: "//*[text()[contains(.,'Löschen')]]",
```

Comment:
Is the deleteTask.feature already adjusted, so that is uses the new tasks page in nuxt?

Reply:
I guess we discussed it already.